.github/labeler.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Labeler
+
+libraries:
+  - lib/**/*
+  - include/grass/*
+  - include/grass/**/*
+  - python/**/*
+
+module:
+  - db/**/*
+  - display/**/*
+  - general/**/*
+  - imagery/**/*
+  - misc/**/*
+  - ps/**/*
+  - raster/**/*
+  - raster3d/**/*
+  - scripts/**/*
+  - temporal/**/*
+  - vector/**/*
+
+# helperscripts:
+#  - utils/*
+#  - utils/**/*
+
+GUI:
+  - gui/**/*
+
+Windows:
+  - mswindows/**/*
+
+macOS:
+  - macosx/**/*
+
+Linux:
+  - rpm/**/*
+  - singularity/**/*
```

Comment:
Hm, docker or Linux label. I don't know.

Reply:
rpm could probably be skipped as it has never (?) received any reports in the past 10 years.